book/src/api.md

```diff
@@ -0,0 +1,1055 @@
+# API Documentation
+
```

Comment:
Can you add a sub-heading here for VolumeSnapshot?  We can add other CRDs later.

Reply:
The tool already generates subheadings for all the included CRDs like VolumeSnapshot, VolumeSnaphotContent and VolumeSnapshotClass:

![image](https://user-images.githubusercontent.com/1330522/163304478-72aac1fb-e793-47c6-9ab7-48966ca7da65.png)

![image](https://user-images.githubusercontent.com/1330522/163304547-0da80e59-ae63-4ce5-b654-db0fa9aaca9d.png)

![image](https://user-images.githubusercontent.com/1330522/163304573-57289bd7-ae4f-4782-83e5-133cea703af6.png)

To confirm my understanding, you would like me to add specifically VolumeSnapshot as a 2nd level subheading?

Reply:
I think I see what you mean. There are other non-snapshot CRDs in CSI. Yes, I can group all these snapshot CRDs under VolumeSnapshot.